core/field.js

```diff
@@ -52,6 +52,52 @@ Blockly.Field = function(text, opt_validator) {
   this.setValidator(opt_validator);
 };
 
+/**
+ * The set of all registered fields, keyed by field type as used in the JSON
+ * definition of a block.
+ * @private
+ */
+Blockly.Field.ALL_ = {};
+
+/**
+ * Registers a field type. May also override an existing field type.
+ * Blockly.Field.fromJson uses this registry to find the appropriate field.
+ * @param {!string} type The field type name as used in the JSON definition.
+ * @param {!{fromJson: Function}} fieldClass The field class containing a
+ *                                 fromJson function that can construct an
```

Comment:
Nit: indent line continuations +4 spaces for these descriptions (here and throughout).

Reply:
So 4 spaces between * and fromJson?

    * @param {!{fromJson: Function}} fieldClass The field class containing a
    *    fromJson function that can construct an ...

Reply:
5 spaces between * and `fromJson`, because there was already one space between * and `@param`.
```
* @param {!{fromJson: Function}} fieldClass The field class containing a
*     fromJson function that can construct an ...
```

Reply:
Ignore me, i see from other docs what you mean!